README.md

```diff
@@ -297,9 +297,9 @@ Here are two simple examples:
 
 ```json
 ...
-"event-expression": "trigger.equals('testtrigger')"
+"event-expression": "trigger.name.equals('testtrigger')"
```

Comment:
Why we event need the `trigger` prefix? I think it can be avoided. Is it possible to avoid using `equals` it is too Java and we want to be language agnostic. WDYT?